File: backend/health-data-service/src/main/java/com/health/data/dto/HealthMetricsDto.java
package com.health.data.dto;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class HealthMetricsDto {
    private String userId;
    private float heartRate;
    private float accX;
    private float accY;
    private float accZ;
    private float spo2;
    private float bodyTemperature;
    private long timestamp;
}

File: app/src/main/java/com/health/healthmonitorwearapp/network/HealthMetricsDto.kt
package com.health.healthmonitorwearapp.network

data class HealthMetricsDto(
    val userId: String,
    val heartRate: Float,
    val accX: Float,
    val accY: Float,
    val accZ: Float,
    val spo2: Float,
    val bodyTemperature: Float,
    val timestamp: Long = System.currentTimeMillis()
)

